src/pages/news-detail/news-detail.js

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import parse from 'html-react-parser';
+import ReactPlayer from 'react-player';
+import {
+  Card,
+  Typography,
+  CardContent,
+  CardHeader,
+  CardMedia
+} from '@material-ui/core';
+import { getArticle } from '../../redux/news-detail/news-detail.actions';
+import { useStyles } from './news-detail.style';
+import { LANGUAGE, TIME_OPTIONS } from '../../configs';
+
+const NewsDetailPage = ({ match, item, getArticle }) => {
```

Comment:
Please, put it inside the `news` folder.

Reply:
I made everything as different pages and separated its logic